<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="30" total="31" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="50890" started-at="2019-10-18T12:05:35Z" finished-at="2019-10-18T12:06:26Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="50890" started-at="2019-10-18T12:05:35Z" finished-at="2019-10-18T12:06:26Z">
      <class name="com.staff.qa.testcases.ConsultantPageTest">
        <test-method status="PASS" signature="setExtentReport(org.testng.ITestContext)[pri:0, instance:com.staff.qa.testcases.ConsultantPageTest@553f17c]" name="setExtentReport" is-config="true" duration-ms="175" started-at="2019-10-18T12:05:35Z" finished-at="2019-10-18T12:05:36Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@77b21474]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setExtentReport -->
        <test-method status="PASS" signature="setupBeforeSuite(org.testng.ITestContext)[pri:0, instance:com.staff.qa.testcases.ConsultantPageTest@553f17c]" name="setupBeforeSuite" is-config="true" duration-ms="232" started-at="2019-10-18T12:05:36Z" finished-at="2019-10-18T12:05:36Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@77b21474]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupBeforeSuite -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.staff.qa.testcases.ConsultantPageTest@553f17c]" name="setUp" is-config="true" duration-ms="7094" started-at="2019-10-18T12:05:36Z" finished-at="2019-10-18T12:05:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="verifySetTheStatusToConsultant()[pri:28, instance:com.staff.qa.testcases.ConsultantPageTest@553f17c]" name="verifySetTheStatusToConsultant" duration-ms="41695" started-at="2019-10-18T12:05:43Z" finished-at="2019-10-18T12:06:25Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.fail(Assert.java:103)
at com.staff.qa.testcases.ConsultantPageTest.verifySetTheStatusToConsultant(ConsultantPageTest.java:1383)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifySetTheStatusToConsultant -->
        <test-method status="PASS" signature="getResult(org.testng.ITestResult)[pri:0, instance:com.staff.qa.testcases.ConsultantPageTest@553f17c]" name="getResult" is-config="true" duration-ms="1366" started-at="2019-10-18T12:06:25Z" finished-at="2019-10-18T12:06:26Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifySetTheStatusToConsultant status=FAILURE method=ConsultantPageTest.verifySetTheStatusToConsultant()[pri:28, instance:com.staff.qa.testcases.ConsultantPageTest@553f17c] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method status="PASS" signature="setupAfterSuite()[pri:0, instance:com.staff.qa.testcases.ConsultantPageTest@553f17c]" name="setupAfterSuite" is-config="true" duration-ms="76" started-at="2019-10-18T12:06:26Z" finished-at="2019-10-18T12:06:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupAfterSuite -->
        <test-method status="PASS" signature="endReport()[pri:0, instance:com.staff.qa.testcases.ConsultantPageTest@553f17c]" name="endReport" is-config="true" duration-ms="236" started-at="2019-10-18T12:06:26Z" finished-at="2019-10-18T12:06:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endReport -->
      </class> <!-- com.staff.qa.testcases.ConsultantPageTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
